### ISSUE TITLE
Configurable TimeOut (TTL) on LBaaS

### ISSUE THREAD

**Issue by mhurtrel** [ovh/public-cloud-roadmap#219]
As a user of load balancer
I want to be able to configure the TTL on load balancer
So that request that require more than the current standard TLL (180 seconds) are handled if I want so

**Comment by tanandy** [comment]
We expect at least to set 30/60 min for idle timeout.

It could be also interesting to enable/disable TCP RESET on LB

**Comment by matmicro** [comment]
Can we expect this to be released asap ?
I agree with the ability to set at least 60min

Does LBaaS concern also Kubernetes LB ? I urgently need this with MKS Loadbalancers.

**Comment by mhurtrel** [comment]
Hi @matmicro We will not develop the feature on the current implementation of MKS loadbalancer, but this will be supported as we will move to Octavia, early 2023 https://docs.openstack.org/octavia/ussuri/configuration/configref.html

**Comment by matmicro** [comment]
Ok so this means that the Octavia LB will be the default one created from Kubernetes.
So as a MKS user it will be transparent for me comparing to the current solution ?
I mean that i am currently using a LB on MKS, in Q1 2023 it will be migrated to Octavia LB automatically (keeping same IP address ?), and my current K8s config files will automatically update the octavia LB ?

So the flavor (S, M, L) will also be available from Kubernetes YAML config file after migration.
Am I right ?

**Comment by mhurtrel** [comment]
@matmicro I confirm Octavia will be the default LB for new and existing clusters (potentially at different points in time).
We have not yet the exact details about the migration for existing cluster, but we will detail this as soon as possible.

I confirm the multiple octavia sizes/flavors will be available and supported through a LB ressource parameter.

**Comment by JacquesMrz** [comment]
@all,

About TTL configuration, I confirm it will be possible to set custom TTL while creating Listeners.
```
--timeout-member-connect <timeout>
Backend member connection timeout in milliseconds. Default: 5000.
--timeout-member-data <timeout>
Backend member inactivity timeout in milliseconds. Default: 50000.
```

And
```
--timeout-member-data <timeout>
Backend member inactivity timeout in milliseconds. Default: 50000.
```

**Comment by clemensgg** [comment]
Hi all i'm currently having real issues to deploy a loadbalancer service on ovh managed kubernetes. 
```
Warning  SyncLoadBalancerFailed  103s               service-controller  Error syncing load balancer: failed to ensure load balancer: error waiting for load balancer to be active: load balancer creation for "xxx-redacted-xxxxxx-xxx" timed out
  Normal   EnsuringLoadBalancer    98s (x2 over 11m)  service-controller  Ensuring load balancer
```
All loadbalancer instances in the OVH frontend stay in status `ACTIVATING` for many hours, external ip status of the respective service stays `pending`

**Comment by mhurtrel** [comment]
Hi @clemensgg I invite you to open a support case, so that our team quickly look at your situation (this githib repo is focus on new feature priorization)

**Comment by yomovh** [comment]
This feature is built in the Public Cloud LoadBalancer so I propose to close this issue.

**Comment by yomovh** [comment]
This will be implemented in #104 so wait for the beta to have those information ;-)
